internal/provider/provider.go

```diff
@@ -88,15 +88,6 @@ func (p *PrefectProvider) Configure(ctx context.Context, req provider.ConfigureR
 		)
 	}
 
-	if config.APIKey.IsUnknown() {
```

Comment:
this is a duplicate block of the lines right above it - prob from a past copy pasta